AndroidManifest.xml

```diff
@@ -143,7 +145,7 @@
             android:name=".authentication.AuthenticatorActivity"
             android:exported="true"
             android:launchMode="singleTask"
-            android:theme="@style/Theme.ownCloud.noActionBar" >
+            android:theme="@style/Theme.ownCloud.Toolbar" >
```

Comment:
You got it right in the first place, There IS a toolbar now ;) (will have to check for toolbar on login screen though)